microprofile-open-api/src/main/java/org/jboss/eap/qe/microprofile/openapi/apps/routing/router/services/DistrictServiceClient.java

```diff
@@ -0,0 +1,53 @@
+package org.jboss.eap.qe.microprofile.openapi.apps.routing.router.services;
+
+import javax.ws.rs.*;
```

Comment:
Please do not use wildcard imports.

Reply:
Right, it must be my IDE that started doing this due to some kind of settings I've changed. Fixing.